js/main.js

```diff
@@ -26,7 +26,8 @@ jQuery(function($) {
 				var $this = $(this);
 
 				// add a tab head link for each tab
-				tabsHead.append('<a href="#">' + $this.find('.gsection_title').html() + '</a>');
+				// add the id as well so gf conditional sections still works
+				tabsHead.append('<a id="' + $this[0].id + '" href="#">' + $this.find('.gsection_title').html() + '</a>');
```

Comment:
Could we rewrite `$this[0].id` in a safer way, for example `$this.attr('id')` (untested)?

Reply:
@gubbigubbi you just commit the changes to your branch and they will be added to your PR

Reply:
Hey @mastef I have commited the changes to my branch. Please feel free to merge :)

Reply:
@gubbigubbi did you also push to the remote? I'm not seeing any recent changes here.

Reply:
@gubbigubbi you need to `git push` them now